### PR TITLE
PD-380 Update Cluster Articles

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit adds the new trueimage shortcode to clustering articles.

It adds information on supported cluster types, requirement to completely/properly remove gluster metadata from a server if it was in another cluster and before you can reuse it in a new cluster, and requirement that all TrueNAS systems must have the same release version.

This commit merges into a new branch for TC 2.3.3 release documentation.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/_includes/ClusterWarnings.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-SMB clusters created in TrueNAS SCALE Bluefin are not  available for cluster expansion. Beginning with TrueNAS SCALE Cobia, we plan to implement a method to enable new volumes for SMB cluster expansion.
+SMB clusters created in TrueNAS SCALE Bluefin are not  available for cluster expansion.
```
Comment (Mrt134) [inline content/_includes/ClusterWarnings.md:1]:
```suggestion
SMB clusters created in TrueNAS SCALE Bluefin are not available for cluster expansion. 
```

File: content/_includes/ClusterWarnings.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-SMB clusters created in TrueNAS SCALE Bluefin are not  available for cluster expansion. Beginning with TrueNAS SCALE Cobia, we plan to implement a method to enable new volumes for SMB cluster expansion.
+SMB clusters created in TrueNAS SCALE Bluefin are not  available for cluster expansion. 
+Beginning with TrueNAS SCALE Cobia, we plan to implement a method to enable new volumes for SMB cluster expansion.
```
Comment (Mrt134) [inline content/_includes/ClusterWarnings.md:2]:
```suggestion
TrueNAS SCALE Cobia plans to implement a method to enable new volumes for SMB cluster expansion.
```

File: content/_includes/ClusterWarnings.md [hunk 3]
```diff
@@ -9,7 +10,15 @@ Configuring or managing clusters within the TrueNAS SCALE UI or Shell can result
 
 Using the clustering feature on a SCALE system adds some restrictions to that system:
 
-* Existing non-clustered SMB shares no longer function.
 * You cannot create new SMB shares separately from the clustering settings.
 * You cannot add the system to a different cluster.
-* Removing single systems from one cluster and migrating to another is not currently supported. Removing a system from a cluster requires deleting the entire cluster.
\ No newline at end of file
+* You cannot remove single systems from one cluster and migrate to another. This is not currently supported.
```
Comment (Mrt134) [inline content/_includes/ClusterWarnings.md:15]:
```suggestion
* Removing or migrating systems from a cluster requires deleting the entire cluster.
```

File: content/_includes/ClusterWarnings.md [hunk 4]
```diff
@@ -9,7 +10,15 @@ Configuring or managing clusters within the TrueNAS SCALE UI or Shell can result
 
 Using the clustering feature on a SCALE system adds some restrictions to that system:
 
-* Existing non-clustered SMB shares no longer function.
 * You cannot create new SMB shares separately from the clustering settings.
```
Comment (Mrt134) [inline content/_includes/ClusterWarnings.md:13]:
```suggestion
* Activating clustering disables individual SMB share creation on cluster member TrueNAS systems.
```

File: content/_includes/ClusterWarnings.md [hunk 5]
```diff
@@ -9,7 +10,15 @@ Configuring or managing clusters within the TrueNAS SCALE UI or Shell can result
...
 * You cannot add the system to a different cluster.
```
Comment (Mrt134) [inline content/_includes/ClusterWarnings.md:14]:
```suggestion
* Systems join a single cluster at a time.
```

File: content/_includes/ClusterWarnings.md [hunk 6]
```diff
@@ -9,7 +10,15 @@ Configuring or managing clusters within the TrueNAS SCALE UI or Shell can result
...
-* Removing single systems from one cluster and migrating to another is not currently supported. Removing a system from a cluster requires deleting the entire cluster.
\ No newline at end of file
+* You cannot remove single systems from one cluster and migrate to another. This is not currently supported. 
+* You cannot remove a system from a cluster until you delete the entire cluster.
```
Comment (Mrt134) [inline content/_includes/ClusterWarnings.md:16]:
```suggestion
```

File: content/_includes/ClusterWarnings.md [hunk 7]
```diff
@@ -9,7 +10,15 @@ Configuring or managing clusters within the TrueNAS SCALE UI or Shell can result
...
 Using the clustering feature on a SCALE system adds some restrictions to that system:
 
-* Existing non-clustered SMB shares no longer function.
 * You cannot create new SMB shares separately from the clustering settings.
 * You cannot add the system to a different cluster.
-* Removing single systems from one cluster and migrating to another is not currently supported. Removing a system from a cluster requires deleting the entire cluster.
\ No newline at end of file
+* You cannot remove single systems from one cluster and migrate to another. This is not currently supported. 
+* You cannot remove a system from a cluster until you delete the entire cluster. 
+* You cannot reuse an existing node without properly leaving an the cluster and cleaning up old gluster-related metadata.
```
Comment (Mrt134) [inline content/_includes/ClusterWarnings.md:17]:
```suggestion
```

File: content/_includes/ClusterWarnings.md [hunk 8]
```diff
@@ -9,7 +10,15 @@ Configuring or managing clusters within the TrueNAS SCALE UI or Shell can result
...
+
+Existing non-clustered SMB shares no longer function.
```
Comment (Mrt134) [inline content/_includes/ClusterWarnings.md:19]: Redundant with SMB config disable comment above.
```suggestion
```

**Review by Mrt134 — DISMISSED** [review]
Positive rewording of the cluster warnings and remove some redundant elements.

**Comment by micjohnson777** [comment]
Why does this still have a "changes requested" status when all changes were accepted and committed? Can we have a new status that indicates ready for review or merge again?

**Comment by Mrt134** [comment]
This is due to the stale review not being dismissed. You can open the dropdown with the change requested review and dismiss the stale review before requesting a new review.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**